tests/unit_tests/backends/test_collisions_methods.py

```diff
@@ -207,3 +207,27 @@ def test_scale_prob_for_adaptive_sdm_gamma(
                 )
         np.testing.assert_array_almost_equal(_gamma.to_ndarray(), expected_gamma)
         np.testing.assert_array_equal(_n_substep, np.asarray(expected_n_substep))
+
+    @staticmethod
+    def test_cell_caretaker(backend_class):
+        # Arrange
+        backend = backend_class()
+        idx = [0, 3, 2, 4]
+
+        cell_start = backend.Storage.from_ndarray(np.asarray([0, 4]))
```

Comment:
do I get it right that cell_start is an output array here, and no initial values are needed?